plugins/linstor.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: linstor
+spec:
+  version: v0.1.2
+  homepage: https://github.com/piraeusdatastore/kubectl-linstor
+  shortDescription: A plugin to control a LINSTOR cluster using kubectl
```

Comment:
A lot of redundant words here. It's clear this is a plugin and it's clear it uses kubectl. 

It's better if you explain what the software does. Please look at the plugin list for examples.